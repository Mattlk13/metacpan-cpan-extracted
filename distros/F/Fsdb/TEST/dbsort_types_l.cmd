prog='dbsort'
args='-n cid'
cmp='diff -c -b '
in=TEST/dbsort_types.in
