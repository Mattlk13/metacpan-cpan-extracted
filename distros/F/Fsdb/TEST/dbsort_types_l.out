#fsdb cid:l cname:a
10	pascal
11	numanal
12	os
#   | dbsort -n cid
